CMake of at least 3.0 is required

This is going to be increasingly rare, I know, but systems with old versions of CMake will produce the following error:

```
$ cmake -DCMAKE_INSTALL_PREFIX=/usr/local/stow/ttyd-1.4.0 ..
CMake Error at CMakeLists.txt:2 (cmake_policy):
  Policy "CMP0048" is not known to this version of CMake.


CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
Missing variable is:
CMAKE_VERSION_COMPILER_ENV_VAR
CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
Missing variable is:
CMAKE_VERSION_COMPILER
CMake Error: Could not find cmake module file: /users/username/devel/web/ttyd/build/CMakeFiles/2.8.12.2/CMakeVERSIONCompiler.cmake
CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
Missing variable is:
CMAKE_1.4.0_COMPILER_ENV_VAR
CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
Missing variable is:
CMAKE_1.4.0_COMPILER
CMake Error: Could not find cmake module file: /users/username/devel/web/ttyd/build/CMakeFiles/2.8.12.2/CMake1.4.0Compiler.cmake
CMake Error: Could not find cmake module file: CMakeVERSIONInformation.cmake
CMake Error: Could not find cmake module file: CMake1.4.0Information.cmake
CMake Error: CMAKE_VERSION_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_1.4.0_COMPILER not set, after EnableLanguage
-- checking for module 'json-c'
--   found json-c, version 0.11
-- Found JSON-C: /lib64/libjson-c.so
-- Configuring incomplete, errors occurred!
See also "/users/username/devel/web/ttyd/build/CMakeFiles/CMakeOutput.log".
See also "/users/username/devel/web/ttyd/build/CMakeFiles/CMakeError.log".
```

A pull request to address this in the README is forthcoming.

--- a/README.md
+++ b/README.md
@@ -45,7 +45,9 @@ brew install ttyd
     make && make install
     ```
 
-    You may need to compile/install [libwebsockets][2] from source if the `libwebsockets-dev` package is outdated.
+    [CMake][19] of at least 3.0 is required. If you get the error message 'Policy "CMP0048" is not known to this version of CMake,' that means you have an old CMake.
+    
+    You may also need to compile/install [libwebsockets][2] from source if the `libwebsockets-dev` package is outdated.
 
 ## Install on Windows
 
@@ -186,3 +188,4 @@ Docker containers are jailed environments which are more secure, this is useful 
   [16]: https://en.wikipedia.org/wiki/ZMODEM
   [17]: https://ohse.de/uwe/software/lrzsz.html
   [18]: https://en.wikipedia.org/wiki/CJK_characters
+  [19]: https://cmake.org/

